exercises/acronym/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include <ctype.h>
+#include <stdlib.h>
+
+static int count_words(const char phrase[]);
+static int is_first_letter_of_word(int c, int b);
```

Comment:
You can pretty easily reorganize your functions to avoid static prototypes. This makes your life easier because it reduces the duplication (and thus maintenance cost), but also introduces an ordering that readers can use when examining your source code (code that is lower in the file is generally at a higher level).